qir/qat/DivisionByZeroPass/DivisionByZeroPass.cpp

```diff
@@ -0,0 +1,152 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "qir/qat/DivisionByZeroPass/DivisionByZeroPass.hpp"
```

Comment:
If you need to change the names of the directories or move the files to different dirs, then you'll have to change the source code (this line).
Consider minimizing the use of the paths in the source code. Consider passing the paths (to the headers) as the command-line args to the compiler (`-Iqir/qat/DivisionByZeroPass`) and not using the paths in the source:
```suggestion
#include "DivisionByZeroPass.hpp"
```

Reply:
There are both pros and cons to using repo absolute paths. The downside is, as you mention, that refactoring becomes slightly harder. The upside is that it removes ambiguity around which file is used. I.e. if two passes both have a file that is called `Config.hpp` then doing `-Iqir/qat/Pass1 -Iqir/qat/Pass2` will confuse which file is being included and you would consequently need longer filenames to resolve the matter. For this reason, I feel reluctant to make this change.